### PR TITLE
Major update V3

### PR DESCRIPTION
**Pull request by Sierd**
Solver, vegetation, Landforms

### CODE REVIEW

File: aeolis/tests/test_model.py [hunk 1]
```diff
@@ -100,8 +100,8 @@ class TestAeoLiSRunner:
 
     def test_parse_callback_from_module(self):
         """Test if the callback function can be loaded from a file"""
-        runner = AeoLiSRunner("aeolis/tests/aeolis.txt")
-        callback = runner.parse_callback("aeolis/tests/callback_example.py:mock_callback")
+        runner = AeoLiSRunner("aeolis.txt")
```
Comment (niketagrawal) [inline aeolis/tests/test_model.py:103]: This path to aeolis.txt file is incorrect. The correct path is "aeolis/test/aeolis.txt"

File: aeolis/tests/test_model.py [hunk 2]
```diff
@@ -100,8 +100,8 @@ class TestAeoLiSRunner:
...
+        callback = runner.parse_callback("callback_example.py:mock_callback")
```
Comment (niketagrawal) [inline aeolis/tests/test_model.py:104]: This path to the callback function is incorrect. The correct path is "aeolis/tests/callback_example.py:mock_callback"

**Review by niketagrawal — CHANGES_REQUESTED** [review]
The path to the aeolis.txt file and the callback function used by the test function is incorrect.

**Comment by Sierd** [comment]
rushed pre-release to kickstart 2024 sprint session

**Pull request merged**